src/index.js

```diff
@@ -45,22 +44,40 @@ removeBtn.forEach((btn) => {
   });
 });
 
+// Store checked status in localStorage
 /* eslint-disable no-plusplus */
 const boxes = document.getElementsByClassName('entertodo').length;
-function save() {
+export default function statusManager(id) {
   for (let i = 1; i <= boxes; i++) {
     const checkbox = document.getElementById(String(i));
     localStorage.setItem(`checkbox${String(i)}`, checkbox.checked);
+
+    const getTodos = getLocalStorage().localTodos;
+    const completedTasks = getTodos.filter((task) => task.index === parseInt(id, 10));
+    completedTasks[0].complete = true;
+    getTodos[id - 1].complete = completedTasks[0].complete;
+    localStorage.setItem('todolist', JSON.stringify(getTodos));
   }
 }
 
-// Loading checkbox status
+// Render checkbox status to UI
 for (let i = 1; i <= boxes; i++) {
-  const checked = JSON.parse(localStorage.getItem(`checkbox${String(i)}`));
-  document.getElementById(String(i)).checked = checked;
+  const getTodos = getLocalStorage().localTodos;
+  if (getTodos[i - 1].complete === true) {
+    const checked = JSON.parse(localStorage.getItem(`checkbox${String(i)}`));
+    document.getElementById(String(i)).checked = checked;
+  }
 }
```

Comment:
- **[OPTIONAL]** You seem to have implemented line-through for checked boxes. It would be nice to add it on page refresh also

Reply:
I will surely implement that. Thank you so much.